### PR TITLE
cdap e2e framework snapshot version fix

### PR DESCRIPTION
**Pull request by rahuldash171**

### CODE REVIEW

**Review by Vipinofficial11 — APPROVED** [review]

**Review by Vipinofficial11 — APPROVED** [review]

**Review by itsankit-google — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -2767,7 +2767,7 @@
         <module>cdap-distributions</module>
         <module>cdap-test</module>
         <module>cdap-integration-test</module>
-        <module>cdap-ui</module>
+<!--        <module>cdap-ui</module>-->
```
Comment (itsankit-google) [inline pom.xml]: why this change?

**Review by itsankit-google — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (rahuldash171) [reply pom.xml]: HI ankit , It was a change related to cdap ui folder which does not have POM file but it was not necessary for this commit , i have removed the change pushed again . Please check and merge this PR .

**Review by itsankit-google — APPROVED** [review]
